jffs2/libjffs2.c

```diff
@@ -530,7 +550,7 @@ static int jffs2_srv_create(jffs2_partition_t *p, oid_t *dir, const char *name,
 	dentry = malloc(sizeof(struct dentry));
 	memset(dentry, 0, sizeof(struct dentry));
```

Comment:
Besides PR.
No checking of the value returned by the `malloc()` function.
Maybe instead  `malloc(), memset()` use `calloc()`.